src/Cms/AppErrors.php

```diff
@@ -111,10 +111,14 @@ protected function handleJsonErrors(): void
                 $httpCode = $exception->getHttpCode();
                 $code     = $exception->getCode();
                 $details  = $exception->getDetails();
-            } else {
+            } elseif (is_a($exception, '\Exception') === true) {
                 $httpCode = 500;
                 $code     = $exception->getCode();
                 $details  = null;
+            } else {
+                $httpCode = 500;
+                $code     = 500;
+                $details  = null;
```

Comment:
It was giving an error in tests without an `Exception` instance. I made an improvement. I want you to check it out.

Reply:
I've also caught all Throwables here